seed/views/portfoliomanager.py

```diff
@@ -96,8 +104,11 @@ def report(self, request):
             properties = content_object['report']['informationAndMetrics']['row']
         except KeyError:
             return JsonResponse(
-                {'status': 'error', 'message': 'Template XML was properly formatted but missing expected keys.'},
-                status=500
+                {
+                    'status': 'error', 'message':
+                    'Processed template successfully, but missing keys -- is the template empty on Portfolio Manager?'
+                },
+                status=status.HTTP_400_BAD_REQUEST
```

Comment:
Cleaned up the verbiage and the response status code (400 is more appropriate here than 500...) to better indicate the response status from PM.